inception-ui-search/src/main/java/de/tudarmstadt/ukp/inception/app/ui/search/sidebar/SearchAnnotationSidebar.java

```diff
@@ -18,20 +18,22 @@
 package de.tudarmstadt.ukp.inception.app.ui.search.sidebar;
 
 import static de.tudarmstadt.ukp.clarin.webanno.api.annotation.util.WebAnnoCasUtil.getAddr;
+import static de.tudarmstadt.ukp.clarin.webanno.api.annotation.util.WebAnnoCasUtil.selectSingleFsAt;
```

Comment:
Many changes here... how are they related to the refactoring?

Reply:
I want to split the loop as much as possible to reduce the complexity of understanding the code, but this is too mechanical, but it has a negative impact on the comprehensibility of the code.